### PR TITLE
Activate realtime in terminus schedules

### PR DESCRIPTION
**Pull request by kadhikari**
https://jira.kisio.org/browse/NAVP-1554

### CODE REVIEW

File: source/jormungandr/jormungandr/schedule.py [hunk 1]
```diff
@@ -342,7 +342,36 @@ def sorter(p):
...
+        for terminus_schedule in resp.terminus_schedules:
+            route_point = _get_route_point_from_stop_schedule(terminus_schedule)
+            rt_proxy = self._get_realtime_proxy(route_point)
+            if rt_proxy:
+                futures.append(pool.spawn(worker, rt_proxy, route_point, request, terminus_schedule))
+
+        for future in gevent.iwait(futures):
+            rt_proxy, terminus_schedule, next_rt_passages = future.get()
+            rt_proxy._update_stop_schedule(terminus_schedule, next_rt_passages)
+        return resp
```
Comment (nonifier) [inline source/jormungandr/jormungandr/schedule.py]: This is a copy and paste from `departure_boards`. Could you please move this into separate functions and re-use the code between `terminus_schedule` and `departure_boards` ?

File: source/time_tables/departure_boards.cpp [hunk 2]
```diff
@@ -454,6 +454,7 @@ void terminus_schedules(PbCreator& pb_creator,
                         const uint32_t depth,
                         const uint32_t count,
                         const uint32_t start_page,
+                        const type::RTLevel rt_level,
```
Comment (nonifier) [inline source/time_tables/departure_boards.cpp:459]: :+1:

**Review by nonifier — APPROVED** [review]
Much better :)

**Review by pbench — APPROVED** [review]
Nice work ! :+1: With lots of tests :1st_place_medal:

**Pull request merged**